CryptoPrices/Resources/Assets.xcassets/ic_arrow_up_green.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "ic_arrow_up_green.pdf",
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
```

Comment:
@Thieurom nice concern you raised here, your suggestion is originally what my approach for handling images assets 👍 Unfortunately, I don't think the assets setup for an SPM module package is easy and I was unable to set it up properly and in the end, the assets placed in the SPM module couldn't be loaded up and displayed on the screen. 

Here is some research that I tried: [here](https://stackoverflow.com/questions/39815054/how-to-include-assets-resources-in-a-swift-package-manager-library), [here](https://stackoverflow.com/questions/73658421/main-bundle-assets-in-swift-package-previews), and even the Apple document [here](https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package) but there is not a proper guide for handling images assets.

I also did come across a seemingly useful [article](https://www.enekoalonso.com/articles/displaying-images-in-swiftui-views-from-swift-package-resources) but eventually, the same problem occurs. 😅

=> If you happen to know any better approach that works properly, please share it 🙇‍♂️ Otherwise, for assets management, we will need to place in the main application like now 😅